templates/docs/layouts/documentation.md

```diff
@@ -0,0 +1,168 @@
+---
+wrapper_template: '_layouts/docs.html'
+context:
+  title: Documentation | Layouts
+---
+
+<style>
+
+.u-thumb-row {
+  --grid-gap: 0.5rem;
+  display: grid;
+  grid-gap: var(--grid-gap);
```

Comment:
I feel like we are overcomplicating this. 
We don't have to make it this specific (to literally repeat the grid implementation). Also, we probably don't want to use CSS grid because it will not work in older browsers making this page just look broken and even more confusing.

If we want this to be HTML, the CSS should be really minimal.

Or it should be an image. Maybe SVG if we want better scaling and font rendering.

All we need it couple boxes with labels